packages/main/src/MessageStrip.js

```diff
@@ -91,6 +72,23 @@ const metadata = {
 		"default": {
 			type: Node,
 		},
+
+		/**
+		 * Defines the icon to be displayed as graphical element within the <code>ui5-messagestrip</code>.
+		 * <br><br>
+		 * <b>Note:</b> If no icon is given, the default icon for the <code>ui5-messagestrip</code> type will be used.
+		 * The SAP-icons font provides numerous options.
+		 * <br><br>
+		 *
+		 * See all the available icons in the <ui5-link target="_blank" href="https://openui5.hana.ondemand.com/test-resources/sap/m/demokit/iconExplorer/webapp/index.html" class="api-table-content-cell-link">Icon Explorer</ui5-link>.
+		 *
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		"icon": {
```

Comment:
If the user can pass everything, we can think of a more generic name. Something like graphicalContent or visualContent

Reply:
I think `icon` is fine, because it is consistent with all other components. It implies that you are expected to pass an icon normally, but as it is HTMLElement, you can pass anything technically.